portal/templates/dashboard/_macros/dashboard-facilities-table.njk

```diff
@@ -25,7 +25,7 @@
       {% endif %}
     {% endset %}
 
-    {% if facility.hasBeenIssued %}
+    {% if facility.hasBeenIssued and facility.hasBeenIssuedAndAcknowledged %}
```

Comment:
Are we sure we want this - last week had a requirement change where an Issued facility should show as Issued on the BSS deal page, before it submitted to UKEF?

Reply:
@IgnatG

Reply:
interesting!
@Zainzzkk  can you clarify with Clara?